docs/Documentation/Development/Libraries/hsl.md

```diff
@@ -79,19 +79,19 @@ cd ${MYLIB}
 ln -s ${METIS_LIBRARY}/libmetis.so
 ```
 
-This has a couple of advantages.  First, the `coinbrew` build will automatically add the `${MYLIB}` directory to the rpath of all constructed libraries and executables.  This means that we don't need to add `${MYLIB}` to the LD_LIBRARY_PATH.  The other advantage is that anaconda puts all the environments libraries and include files in the same directories with `libmetis.so` and `metis.h`.  Many of these libraries overlap with those used by HSL, Mumps and Ipopt but are not necessarily the same versions.  Loading a different version of a library than those compiled against can cause unexpected behavior.
+This has a couple of advantages.  First, the `coinbrew` build will automatically add the `${MYLIB}` directory to the rpath of all constructed libraries and executables.  This means that we don't need to add `${MYLIB}` to the LD_LIBRARY_PATH.  The other advantage is that anaconda puts all the environments libraries and include files in the same directories with `libmetis.so` and `metis.h`.  Many of these libraries overlap with those used by HSL, Mumps and IPOPT but are not necessarily the same versions.  Loading a different version of a library than those compiled against can cause unexpected behavior.
 
 #### Configure and Install
 
-Follow the [Ipopt installation instructions here](ipopt.md#eagle) to finish the installation of HSL solvers on Eagle.
+Follow the [IPOPT installation instructions here](../../Applications/ipopt.md#installation) to finish the installation of HSL solvers on Eagle.
 
 ### MacOS
 
 The following installation has been tested on Apple's M1 ARM based processors.
```

Comment:
Why are we including the install instructions for Mac? We wouldn't be able to support this so I don't think it should be included unless it's relevant for usage on HPC

Reply:
This is a little grey area, we sometimes provide support for some apps, e.g., Xpress, on Mac too. But if need be, we can delete it as part of another PR just to keep things tidy.